inz-demo/id-card/Issues/address/description.xml

```diff
@@ -0,0 +1,34 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo ID Card Address</en>
```

Comment:
I would say that we can just call `Demo Address`. I know the issuer is ID Card, but I would say that we are issuing a proof of address from the `id-card` issuer and not an address tied exclusively to the ID Card.

What are your thoughts, @camfindlay ?